SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -3,6 +3,7 @@
 #include "SpatialGDKSettings.h"
 #include "Misc/MessageDialog.h"
 #include "Misc/CommandLine.h"
+#include "Settings/LevelEditorPlaySettings.h"
```

Comment:
Do we need this header include? it is only included with editors below.

Reply:
Looks like a bad merge - it's clearly not needed.

Reply:
yep, bad merge, will remove